ibm-cloud-pak-for-data/azuredeploy.json

```diff
@@ -1072,7 +1132,7 @@
 			}
 		},
 		{
-			"name": "CloudPakDVDeployment",
+			"name": "PreloadImagesDV",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties